src/cmd/cli/command/compose.go

```diff
@@ -410,39 +407,33 @@ func makeComposeConfigCmd() *cobra.Command {
 		Short: "Reads a Compose file and shows the generated config",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			loader := configureLoader(cmd)
-			ctx := cmd.Context()
-			provider, err := getProvider(ctx, loader)
-			if err != nil {
-				return err
-			}
 
+			ctx := cmd.Context()
 			project, loadErr := loader.LoadProject(ctx)
-			if loadErr == nil {
-				_, _, err = cli.ComposeUp(ctx, project, client, provider, compose.UploadModeIgnore, defangv1.DeploymentMode_UNSPECIFIED_MODE)
-				if err != nil && errors.Is(err, cli.ErrDryRun) {
-					return nil
+			if loadErr != nil {
+				if nonInteractive {
+					return loadErr
 				}
 
-				loadErr = err
-			}
-
-			if nonInteractive {
-				return loadErr
-			}
-
-			if loadErr != nil {
-				project, err = CreateProjectForDebug(loader)
+				term.Error("Cannot load project:", loadErr)
+				project, err := createProjectForDebug(loader)
```

Comment:
Do we want to support this ai debugger for any command which loads a project? This makes me wonder if we should fold this fallback for debug into the `LoadProject` function.